Source/DataSources/KmlLookAt.js

```diff
@@ -0,0 +1,70 @@
+define([
+        '../Core/defined',
+        '../Core/defaultValue',
+        '../Core/Math',
+        '../Core/HeadingPitchRange',
+        '../Core/Cartesian3',
+        '../Core/BoundingSphere'
+    ], function(
+        defined,
+        defaultValue,
+        CesiumMath,
+        HeadingPitchRange,
+        Cartesian3,
+        BoundingSphere ) {
+    'use strict';
+
```

Comment:
Whitespace issues throughout this file. Some functions indent with 2 spaces, some with 4.